packages/grid/src/features/standard-table/hooks/UseCellBackground.ts

```diff
@@ -23,14 +23,14 @@ const useBackground = <TItem>(
   ]);
 
 export const useCellBackgroundByColumnId = <T>(columnId: string, item: T) => {
-  const { background, backgroundResolver } = useColumnFromConfig(columnId);
+  const { background, backgroundResolver } = useColumnConfigById(columnId);
   return useBackground(backgroundResolver, item, background);
 };
 
-export const useCellBackgroundByColumnConfig = <TItem>(
-  columnConfig: StandardTableColumnConfig<TItem, {}>,
+export const useCellBackgroundByColumnConfig = <TItem, TItemValue>(
+  columnConfig: StandardTableColumnConfig<TItem, TItemValue> | undefined,
```

Comment:
```suggestion
  columnConfig: StandardTableColumnConfig<TItem, TItemValue> = {},
```

Reply:
This doesn't work, {} is not compatible with StandardTableColumnConfig.